cmd/ike/cmd/develop.go

```diff
@@ -54,7 +54,7 @@ func NewDevelopCmd() *cobra.Command {
 	}
 
 	developCmd.Flags().StringP("deployment", "d", "", "name of the deployment or deployment config")
-	developCmd.Flags().IntP("port", "p", 8000, "port to be exposed")
+	developCmd.Flags().StringP("port", "p", "8000", "port to be exposed in format local[:remote]")
```

Comment:
Might make sense to adjust one of the tests here:

https://github.com/aslakknutsen/istio-workspace/blob/master/cmd/ike/cmd/develop_test.go#L157-L189

This also opens up a question if we want to support multiple ports? If so - I will open an issue for that. See https://github.com/telepresenceio/telepresence/issues/280

Reply:
Yea, multi port makes sense

Reply:
addressed https://github.com/aslakknutsen/istio-workspace/pull/44/commits/d6a8f7c65c249e40a8b3f07a28e4a8e6853ccec9

Reply:
#45